src/ripple/protocol/impl/Feature.cpp

```diff
@@ -102,7 +102,7 @@ detail::supportedAmendments()
         "TickSize",
         "fix1368",
         "Escrow",
-        "CryptoConditionsSuite",
+        // "CryptoConditionsSuite", NO NOT REUSE
```

Comment:
```suggestion
        // "CryptoConditionsSuite", DO NOT REUSE
```

Reply:
Thanks Elliot! Fixed in `10f000c89 [fold] Fix typo
`